Dockerfile

```diff
@@ -32,7 +32,10 @@ RUN npm install --global grunt
 RUN echo '{ "allow_root": true }' > /root/.bowerrc
 RUN cd mbofui && bower install && npm install && grunt docker
 
+# create saml directory for shib files unless running on local machine
+RUN if [ ! -d ./hacks_mbof/saml/ ]; then mkdir ./hacks_mbof/saml/; fi
+
 EXPOSE 8000
 
 # copy settings file and launch django
-CMD cp /usr/share/ocellus/settings.py ./hacks_mbof/; REMOTE_USER=bjensen gunicorn --workers=1 --bind=0.0.0.0:8000 hacks_mbof.wsgi:application
+CMD cp /usr/share/ocellus/settings.py ./hacks_mbof/; cp -a /usr/share/ocellus/saml/. ./hacks_mbof/saml/;REMOTE_USER=bjensen gunicorn --workers=1 --bind=0.0.0.0:8000 hacks_mbof.wsgi:application
```

Comment:
Are you sure you want the `-a` option to `cp` here?  The manual says this option does:

> Preserves structure and attributes of files but not directory structure.

It doesn't seem like a common thing to do.  I just tried an example of this and the less mysterious options `-pr` does the same thing.

Also, the first path you use, `/usr/share/ocellus/saml/.`, has a trailing dot.  Why not use `/usr/share/ocellus/saml/` instead?